### ISSUE TITLE
fitBounds top/bottom paddings are reversed

### ISSUE THREAD

**Issue by NameFILIP** [uber-web/math.gl#301]
I'm passing the following padding to `fitBounds`:

```
padding: {
  bottom: 0,
  left: 0,
  right: 0,
  top: 0 + 50
}
```

But seeing the padding on the bottom:
![image](https://user-images.githubusercontent.com/834796/182674742-614e62bb-dcb0-4f20-9036-cd93c8f259c5.png)

If I specify the `bottom` one then I see it on the top. Why is it reversed?

See the [Codesandbox Example](https://codesandbox.io/s/react-map-gl-maplibre-gl-6zpmur?file=/src/App.js)

Using: `@math.gl/web-mercator: 3.6.2`

**Comment by Pessimistress** [comment]
Published 3.6.3

**Comment by NameFILIP** [comment]
Awesome, thank you for the quick action!
